packages/terra-navigation-side-menu/src/NavigationSideMenu.scss

```diff
@@ -4,4 +4,10 @@
     padding-bottom: var(--terra-navigation-side-menu-list-padding-bottom, 0.125rem);
     padding-top: var(--terra-navigation-side-menu-list-padding-top, 0.125rem);
   }
+
+  .side-menu-action-header {
```

Comment:
These allow us to apply a micro theme to the action header for just this specific location